packages/engine/Source/Scene/BingMapsImageryProvider.js

```diff
@@ -120,7 +120,14 @@ function metadataSuccess(data, imageryProviderBuilder) {
   imageryProviderBuilder.maximumLevel = resource.zoomMax - 1;
   imageryProviderBuilder.imageUrlSubdomains = resource.imageUrlSubdomains;
   imageryProviderBuilder.imageUrlTemplate = resource.imageUrl;
-  imageryProviderBuilder.attributionList = resource.imageryProviders;
+  const validProviders = !defined(resource.imageryProviders)
+    ? resource.imageryProviders
+    : resource.imageryProviders.filter((provider) =>
+        // prevent issues with the imagery API from crashing the viewer when the expected properties are not there
+        // See https://github.com/CesiumGS/cesium/issues/12088
+        provider.coverageAreas?.some((area) => defined(area.bbox))
+      );
+  imageryProviderBuilder.attributionList = validProviders;
```

Comment:
Not trying to bikeshed, but I think this line is getting a bit clunky in terms of readability. Can we simplify? ie

```suggestion
  let validProviders;
  if (defined(resource.imageryProviders)) {
    // prevent issues with the imagery API from crashing the viewer when the expected properties are not there
    // See https://github.com/CesiumGS/cesium/issues/12088
    validProviders = resource.imageryProviders.filter((provider) =>
        provider.coverageAreas?.some((area) => defined(area.bbox))
      );
  }
  imageryProviderBuilder.attributionList = validProviders;
```

Reply:
I don't know if I find it personally more or less readable than what I had but no need to go back and forth. Updated